utilities/run-test-suites/gsheetsImport/resultsImporter.py

```diff
@@ -31,6 +32,26 @@
                "Not Applicable"]
 
 
+def get_range(worksheet_data, start_row, start_col, end_row, end_col):
+    result = []
+    for row_no in range(start_row-1, end_row):
+        row = worksheet_data[row_no] if row_no < len(worksheet_data) else []
+        for col_no in range(start_col-1, end_col):
+            result.append(gspread.Cell(row_no + 1, col_no + 1, row[col_no] if col_no < len(row) else ""))
+    return result
+
+
+def ranges_equal(first, second):
+    if len(first) != len(second):
+        return False
+    for i in range(len(first)):
+        if (first[i].row != second[i].row or
+                first[i].col != second[i].col or
+                first[i].value != second[i].value):
+            return False
+    return True
+
+
 def gsheets_import(test_results, worksheet, filename, start_col=1):
     """Upload results data to spreadsheet"""
 
```

Comment:
I wonder if we should be using `worksheet_get_all_values(ValueRenderOption.FORMULA)` given we're using `'USER_ENTERED'` later?